src/java/org/apache/cassandra/db/CounterMutationVerbHandler.java

```diff
@@ -40,10 +48,18 @@ public void doVerb(final Message<CounterMutation> message)
         logger.trace("Applying forwarded {}", cm);
 
         // Initialize the sensor and set ExecutorLocals
+        Collection<TableMetadata> tables = message.payload.getPartitionUpdates().stream().map(PartitionUpdate::metadata).collect(Collectors.toSet());
         RequestSensors requestSensors = new RequestSensors();
         ExecutorLocals locals = ExecutorLocals.create(requestSensors);
         ExecutorLocals.set(locals);
 
+        // Initialize internode bytes with the inbound message size:
+        tables.forEach(tm -> {
+            Context context = Context.from(tm);
+            requestSensors.registerSensor(context, Type.INTERNODE_BYTES);
+            requestSensors.incrementSensor(context, Type.INTERNODE_BYTES, message.serializedSize(MessagingService.current_version) / tables.size());
```

Comment:
> message.serializedSize

I realized for inbound messages we use the whole message serialized size, should we use the payload size for consistency?

Reply:
+1 https://github.com/datastax/cassandra/pull/1081/commits/8543504b2ac19f841b5700fb444bd3ee943c2edd